quakemigrate/signal/scan.py

```diff
@@ -177,6 +177,9 @@ def __init__(self, archive, lut, onset, run_path, run_name, **kwargs):
             raise util.OnsetTypeError
         self.onset.post_pad = lut.max_traveltime
 
+        self.pre_pad = 0.
+        self.post_pad = 0.
```

Comment:
Note to review this when the `_read_waveform_data` thing has been sorted out and removed from scan. These ideally shouldn't be exposed/we should avoid passing these around via the object.